examples/example-webpack/package.json

```diff
@@ -5,17 +5,36 @@
   "description": "A simple webpack example to test stylexjs/webpack-plugin",
   "main": "index.js",
   "scripts": {
-    "example:build": "webpack",
-    "example:dev": "webpack --watch"
```

Comment:
@necolas I felt having `example:` prefix is bit uncommon for me as a developer so I changed it to dev, serve and build. Is there any intentional purpose behind the prefix?

Reply:
So we don't build the examples in CI: https://github.com/facebook/stylex/blob/main/package.json#L9

Reply:
Will revert it back with example: prefix then